features/cli/deploy.feature

```diff
@@ -347,6 +347,7 @@ Feature: deployment related features
       | replicas | 0                |
     Then the step should succeed
     Given I wait until number of replicas match "0" for replicationController "ab-example-a-1"
+    And I wait until the status of deployment "ab-example-b" becomes :complete
```

Comment:
@zhouying7780 i have seen in multiple cases that this line fails, may be better to add  as below ?

And I wait for the steps to pass
"""
oc get deployment ab-example-b and check ffor status 
"""